### PR TITLE
Added P_4_2_2_01

### PR DESCRIPTION
**Pull request by PaulRaschke**

### CODE REVIEW

File: 01_P/P_4_2_2_01/sketch.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+function setup() {
+  createCanvas(1024, 1024);
+  background(0); 
+
+  // specify a path or use selectInput() to load a video
+  // or simply put it into the data folder
+  var path = "data/video.mp4";
+  img = loadImage("P_4_2_2_01.png");
+  movie = createVideo(path);
```
Comment (Niels-NTG) [inline 01_P/P_4_2_2_01/sketch.js]: Does this even work? It's better to use p5's build in `preload` function for loading the video file.
```javascript
function preload() {
  movie = createVideo('data/video.mp4')
}
```

Such you don't need to listen for a `canplay` event and use a `videoLoaded` flag.

File: 01_P/P_4_2_2_01/sketch.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    }
+
+    currentImage++;
+    if (currentImage >= imageCount) noLoop();
+  }
+}
+
+function keyReleased(){
+  if (key == 's' || key == 'S') saveCanvas(gd.timestamp(), 'png');
+}
```
Comment (Niels-NTG) [inline 01_P/P_4_2_2_01/sketch.js]: Insert an empty newline. That's nicer for git and some editors.

File: 01_P/P_4_2_2_01/sketch.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+ */
+ 
+'use strict';
+
+// horizontal and vertical grid count
+// take care of the aspect ratio ... here 4:3
+var tileCountX = 12;
+var tileCountY = 16;
+var tileWidth, tileHeight;
+var imageCount = tileCountX*tileCountY;
```
Comment (Niels-NTG) [inline 01_P/P_4_2_2_01/sketch.js]: Always add spaces around operators

**Comment by Niels-NTG** [comment]
Video should be re-encoded to be h.264 instead of MPEG-4. The latter has poor browser support. Also strip out the audio, since that doesn't do anything for this sketch.

**Comment by b-g** [comment]
@PaulRaschke Many thanks for this! Welcome to social coding! :) Niels is right ... please adjust and take care of the improvements. (Just commit them to your local feature branch and press sync. The commits will show up under #119. Please ping us here then, so that we can have a 2nd look). Cheers!

**Comment by b-g** [comment]
@PaulRaschke PR closed! As redundant with #120 (did you accidentally add both sketches to the same PR?)

**Pull request closed without merging**